### ISSUE TITLE
MOZILLA ADD-ONS PAGE REMOVED

### ISSUE THREAD

**Issue by Benja-S** [timmyRS/Universal-Bypass#1092]
https://addons.mozilla.org/en-US/firefox/addon/universal-bypass/

Can't install on firefox, is there any alternative link to install?
(without building from source)

Thx in advance

**Comment by timmyRS** [comment]
I've submitted a new version, we'll just have to wait.

**Comment by pin-grid-array** [comment]
Would it be possible for you to explain, in layman's terms, how an average Firefox user could build this extension from the source code? I don't know much about programming, but I can certainly follow directions.

**Comment by timmyRS** [comment]
1. Download current master: https://github.com/timmyRS/Universal-Bypass/archive/master.zip
2. Unpack zip
3. Delete `injection_script.js`
4. Load the folder into Firefox:
![image](https://user-images.githubusercontent.com/11009697/78166728-86bc1b80-744d-11ea-8d91-5e4b3aa549e1.png)
![image](https://user-images.githubusercontent.com/11009697/78166756-90de1a00-744d-11ea-8b63-efe5268b66d6.png)
![image](https://user-images.githubusercontent.com/11009697/78166861-c3881280-744d-11ea-9729-be4e23aafcdd.png)

**Comment by nl255** [comment]
Would you consider uploading an unsigned xpi for Waterfox (or unbranded/nightly Firefox) users until it is available on AMO again?

**Comment by timmyRS** [comment]
https://addons.mozilla.org/firefox/downloads/file/3539342/universal_bypass-13.8.5.1-an+fx.xpi

**Comment by timmyRS** [comment]
For anyone wondering what happened: 4 months ago, Universal Bypass added auto-updating. Now, with Mozilla having few reviewers available, they want me to remove auto-updating, and want a human reviewer to verify that. Absolute geniuses.

**Comment by Sciencentistguy** [comment]
Is there a way you can release a signed xpi for the firefox extension so we don't need to install it every time we open firefox?

**Comment by jsaneu** [comment]
Yo can disable addons verification in [Firefox Nightly](https://nightly.mozilla.org/).

Flag: `xpinstall.signatures.required: false`

**Comment by timmyRS** [comment]
https://addons.mozilla.org/firefox/downloads/file/3539342/universal_bypass-13.9.0.0-an+fx.xpi

**Comment by 3xploiton3** [comment]
> 
> 
> https://addons.mozilla.org/firefox/downloads/file/3539342/universal_bypass-13.9.0.0-an+fx.xpi

links dead,

**Comment by timmyRS** [comment]
We're now self-hosting: https://universal-bypass.org/install
